examples/jsm/tsl/display/TRAANode.js

```diff
@@ -507,10 +507,6 @@ class TRAANode extends TempNode {
 
 			const smoothedOutput = add( currentColor.mul( currentWeight ), clampedHistoryColor.mul( historyWeight ) ).div( max( currentWeight.add( historyWeight ), 0.00001 ) ).toVar();
 
-			// debug: visualize where unoccluded areas have appeared
-
-			//If( rejectPixel, () => { smoothedOutput.assign( vec3( 1.0, 0.0, 0.0 ) ); } );
```

Comment:
Sweet!
It’s tough to tell on my phone without using the Cornell Box scene 🫠
I can check to see if the rejectPixel buffer is being generated properly in 5-6 hours when it’s morning here 😅

You can check earlier by adding this back and seeing if the buffer looks right! @Mugen87